v2/src/lib.rs

```diff
@@ -307,19 +297,26 @@ impl Volume for InfluxDbVolume {
             Some(creds) => creds,
             None => bail!("No credentials specified to access database '{}'", db),
         };
+
         let admin_client = match std::panic::catch_unwind(|| {
             Client::new(url.clone(), &admin_creds.org_id, &admin_creds.token)
         }) {
             Ok(client) => client,
             Err(e) => bail!("Error in creating client for InfluxDBv2 volume: {:?}", e),
         };
 
+        let none_key: OwnedKeyExpr = match OwnedKeyExpr::from_str(NONE_KEY) {
+            Ok(key) => key,
+            Err(e) => bail!("Error in creating client for InfluxDBv2 volume: {:?}", e),
```

Comment:
True, this case should never happen, 
I am thinking more of the case if someone were to change `NONE_KEY` in the future, without updating this block, it would lead to a runtime failure.